warehouse/ingest-core/src/main/java/datawave/util/flag/FlagMaker.java

```diff
@@ -238,9 +237,10 @@ public void run() {
     protected void processFlags() throws IOException {
         FileSystem fs = getHadoopFS();
         log.trace("Querying for files on {}", fs.getUri().toString());
+        
         for (FlagDataTypeConfig fc : fmc.getFlagConfigs()) {
-            long startTime = System.currentTimeMillis();
             String dataName = fc.getDataName();
+            final FlagMetrics metrics = new FlagMetrics(fs, fc.isCollectMetrics());
```

Comment:
We do not want to create one metrics file across all flag files we write.  We need a new one per flag file.  You were correct passing the instance into the write() method, but not correct passing in an instance into the writeFlagFile() method.